scalafmt-tests/src/test/scala/org/scalafmt/util/GitOpsTest.scala

```diff
@@ -37,7 +37,7 @@ class GitOpsTest extends fixture.FunSuite {
 
   def touch(name: String = Random.alphanumeric.take(10).mkString)(
       implicit ops: GitOpsImpl): AbsoluteFile = {
-    val f = File.createTempFile(name, ".ext", ops.workingDirectory.jfile)
+    val f = File.createTempFile(name, ".ext", ops.rootDir.get.jfile)
```

Comment:
If it gets too bad we can just make the `.ext` into something like `.ignoreme` and add that to the `.gitignore`. More than likely this won't be an issue though.